HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -71,6 +71,18 @@ def customiseFor18330(process):
           for pset in producer.vertexCollections:
                if pset.algorithm.value() == "AdaptiveVertexFitter" and not hasattr(pset, "chi2cutoff"):
                     pset.chi2cutoff = cms.double(3.0)
+     return process
+
+
+# Add new parameters required by RecoTauBuilderConePlugin
+def customiseFor18429(process):
+     for producer in producers_by_type(process, "RecoTauProducer"):
+          if hasattr(producer,'builders'):
+               for pset in producer.builders:
+                    if not hasattr(pset,'minAbsPhotonSumPt_insideSignalCone'):
```

Comment:
so this looks to be just a more inefficient version of existsAs... Could this not be replaced by a fillDescriptions instead?